prosemirror-jdita/src/plugin.ts

```diff
@@ -41,14 +82,40 @@ export function shortcuts(schema: Schema) {
   });
 }
 
+/**
+ * Create a new instance of MenuItem and attach a command
+ *
+ * @example of the returned MenuItem
+ * MenuItem {spec: {class: "ic-undo", enable: undo(state, dispatch), icon:{}, run: undo(state, dispatch), title: "Undo"}}
+ *
+ * @param command - The command associated with the menu item
+ * @param props - The properties of the menu item
+ * @returns A new MenuItem instance
+ */
 function commandItem(command: Command, props: Partial<MenuItemSpec> = {}) {
+  // MenuItem: An icon or label that, when clicked, executes a command.
   return new MenuItem({
     ...props,
+    // run: Function to execute when the menu item is activated
     run: command,
+    // enable: Function that is used to determine if the item is enabled.
     enable: command,
```

Comment:
Redundant comment. The code's intention is clear.

Reply:
Fixed with f092276.